unicodetools/src/main/java/org/unicode/xml/GeneratePropertyValues.java

```diff
@@ -1717,11 +1793,20 @@ private static List<String> getDoNotEmitTypeValues() {
         return values;
     }
 
+    private static List<String> getkEHCoreValues() {
+        List<String> values = new ArrayList<>();
+        for (kEH_Core_Values kEHCoreValues : kEH_Core_Values.values()) {
+            values.add(kEHCoreValues.getShortName());
+        }
+        Collections.sort(values);
+        return values;
+    }
+
     // ********************* Utility methods ********************//
 
-    private static HashMap<String, TR38Details> parseTR38() throws IOException, URISyntaxException {
-        HashMap<String, TR38Details> syntaxTR38 = new HashMap<>();
-        URI uri = new URI(TR38URL);
+    private static HashMap<String, TRDetails> parseTR(String Url) throws IOException, URISyntaxException {
```

Comment:
variables in lowerCamelCase...
```suggestion
    private static HashMap<String, TRDetails> parseTR(String url) throws IOException, URISyntaxException {
```

Reply:
Done